velox/row/UnsafeRowFast.cpp

```diff
@@ -1083,9 +1083,13 @@ RowVectorPtr deserializeRows(
       std::vector<char*> nestedData(numRows);
       std::vector<size_t> nestedOffsets(numRows, 0);
       for (auto row = 0; row < numRows; ++row) {
-        const auto offset =
-            readInt32(data[row] + offsets[row] + sizeof(int32_t));
-        nestedData[row] = data[row] + offset;
+        const auto isTopLevelNull =
+            rawNulls != nullptr && bits::isBitNull(rawNulls, row);
```

Comment:
nit: could drop `!= nullptr`.

Reply:
The rawNulls might be null, we can not check `bits::isBitNull` in that case.

Reply:
I assume `rawNulls != nullptr && bits::isBitNull(rawNulls, row)` is equal to `rawNulls && bits::isBitNull(rawNulls, row)`, no?

Reply:
updated, thanks!